libs/template/templates/default-sql/template/{{.project_name}}/src/sample_1.sql.tmpl

```diff
@@ -0,0 +1,4 @@
+-- This query is executed using Databricks Workflows as defined in resources/{{.project_name}}_sql_job.yml.
+
+CREATE OR REPLACE VIEW taxis AS
+SELECT * FROM json.`/databricks-datasets/nyctaxi/sample/json/`
```

Comment:
Is there a way we can configure the catalog/schema for this query?

Reply:
@lennartkats-db Can you comment on this?

Reply:
Sorry, as you might have seen in the Slack thread this capability doesn't exist yet. Instead, customers can configure a default catalog / schema at the workspace level at this point. If they're really eager about this, parameterization can work somewhat, but the IDE plugin doesn't support this well. At this point I'd instead recommend using the `dbt-sql` template to differentiate by target.

Reply:
Thanks -- tbh, I think this template is not all that useful if we can't parameterize the catalog/schema. Without differentiation there is no way to have user isolation in a single dev workspace, nor colocate dev/prod in the same workspace. I would expect to have a catalog/schema parameter at the job level to control these defaults.

Reply:
Discussed offline: we should add some messaging to this template: likely a comment that links to a GitHub issue that tracks our support for a setting the catalog/schema.

Reply:
The new template now includes {{catalog}} as a job parameter. We're not 100% happy with this approach, but it's better than the alternative. A native catalog/schema option is coming to the task type soon. That will make it much cleaner and will make the queries work with the VS Code SQL plugin again. cc @fjakobs